File: Demo/镂空UIView/镂空UIView/ViewController.m
//
//  ViewController.m
//  镂空UIView
//
//  Created by mingcol on 2019/7/8.
//  Copyright © 2019 mingcol. All rights reserved.
//

#import "ViewController.h"
#import "LGDHollowView.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
    
    LGDHollowView *vc = [[LGDHollowView alloc] initWithFrame:self.view.bounds];
    [self.view addSubview:vc];
}


@end

File: Demo/SWIFT/富文本/富文本/ViewController.swift
//
//  ViewController.swift
//  富文本
//
//  Created by LvGuangDa on 2019/9/22.
//  Copyright © 2019 shijunnian. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
        
        let a = UILabel(frame: CGRect(x: 50, y: 100, width: 300, height: 15))
        self.view.addSubview(a)
        
        let attrStr = NSMutableAttributedString.init(string: "01 周五")
        attrStr.addAttribute(NSAttributedString.Key.font, value:UIFont.systemFont(ofSize: 10), range:NSRange.init(location:3, length: 2))
        a.attributedText = attrStr
        
    }

   

}

